Dockerfile

```diff
@@ -7,12 +7,13 @@ ENV HOST 0.0.0.0
 ENV NODE_ENV ${NODE_ENV}
 ENV API_URL ${API_URL}
 
-EXPOSE 3000
 WORKDIR /usr/src/app
-CMD [ "yarn", "start" ]
 
-COPY package.json yarn.lock ./
-RUN yarn install
+COPY package.json yarn.lock nuxt.config.js /usr/src/app/
+COPY src /usr/src/app/src
 
-COPY . /usr/src/app
+RUN yarn
 RUN yarn build
+
+EXPOSE 3000
+CMD [ "yarn", "start" ]
```

Comment:
Changes in this file also needs to be reverted, because their order is on purpose for caching layers (see commits regarding fast docker builds)